test/windows/windows_shim.c

```diff
@@ -104,6 +97,19 @@ pthread_rwlock_unlock(pthread_rwlock_t *rwlock)
 	return (0);
 }
 
+int
+pthread_rwlock_tryrdlock(pthread_rwlock_t *rwlock)
+{
+	return (TryAcquireSRWLockShared(&rwlock->rwlock) ? 0 : EBUSY);
+}
```

Comment:
Can this be removed?

Reply:
It can, but it's reasonable to leave it in, some utility might want it someday?